sharing/src/test/scala/io/delta/sharing/spark/DeltaSharingDataSourceCMSuite.scala

```diff
@@ -0,0 +1,977 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.sharing.spark
+
+import java.io.File
+
+import org.apache.spark.sql.delta.{
+  BatchCDFSchemaEndVersion,
+  BatchCDFSchemaLatest,
+  BatchCDFSchemaLegacy,
+  DeltaUnsupportedOperationException
+}
+import org.apache.spark.sql.delta.sources.DeltaSQLConf
+import org.apache.spark.sql.delta.test.DeltaSQLCommandTest
+import org.apache.spark.sql.delta.sharing.DeltaSharingTestSparkUtils
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.streaming.{StreamingQueryException, StreamTest, Trigger}
+import org.apache.spark.sql.types.{IntegerType, StringType, StructType}
+
+class DeltaSharingDataSourceCMSuite
```

Comment:
why do you need another suite with the tests repeating? can we have the CM suite extend the `DeltaFormatSharingSourceSuite` with `beforeAll` handling the default column mapping settings for created tables? Just like how we have suites in `Delta-Spark`.